### ISSUE TITLE
New release 1.1.0 does not include binaries

### ISSUE THREAD

**Issue by flyaruu** [extism/java-sdk#33]
Hi, I was trying to use the count-vowel example with the new 1.1.0 release, but I could not get it to work, the process seems to just die.

On closer inspection, the 1.1.0 version does not include any binaries. The 1.0.1 version is about 38 Mb, while the 1.1.0 is just 28Kb.

So I guess something went wrong in the build?

**Comment by bhelx** [comment]
I don’t think we ever included them. That was an open issue. Do you mind checking @evacchi

**Comment by flyaruu** [comment]
This one does: https://mvnrepository.com/artifact/org.extism.sdk/extism/1.0.1

**Comment by flyaruu** [comment]
I meant native libraries when I said binaries. If I transplant the native libraries from the latest release (1.8.0) into the 1.1.0 it seems to work alright
